sources/packages/backend/libs/integration/src/integration.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Module } from "@nestjs/common";
+import { IER12IntegrationModule } from "./institution-integration/ier-integration/ier12-integration.module";
+
+@Module({
+  imports: [IER12IntegrationModule],
+})
+export class IntegrationModule {}
```

Comment:
I am not sure about having a module to aggregate all the integrations. I was under the impression that we would have one module per integration and that is it.